runtime/tests/interpreter/reference_test.go

```diff
@@ -696,11 +676,7 @@ func TestInterpretReferenceUseAfterShiftStatementMove(t *testing.T) {
                   }
 
                   fun borrowR2(): &R2? {
-                      let optR2 <- self.r2 <- nil
-                      let r2 <- optR2!
-                      let ref = &r2 as &R2
-                      self.r2 <-! r2
-                      return ref
+                      return &self.r2 as &R2?
```

Comment:
The test code was updated in this test-case to keep supporting the rest of the test.
A duplicate of this test is added below (`"container in account, reference in stack"`) which doesn't change the code, but instead asserts the new behaviour.